### ISSUE TITLE
at-rule-empty-line-before conflicts with prettier and is part of stylelint-config-standard

### ISSUE THREAD

**Issue by felixfbecker** [shannonmoeller/stylelint-config-prettier#4]
Prettier:
```scss
    .navbar-toggler {
      @if (lightness( $navbar-brand-bg ) > 40) {
        color: $navbar-color;
      } @else {
        color: white;
      }
    }
```

After `stylelint --fix`:
```scss
    .navbar-toggler {
      @if (lightness( $navbar-brand-bg ) > 40) {
        color: $navbar-color;
      }

 @else {
        color: white;
      }
    }
```

**Comment by shannonmoeller** [comment]
Will disable. Thanks!

**Comment by yhuard** [comment]
Hi @shannonmoeller, 

First, let me thank you for your preset, I find it really helpful. 

Now, the rule @felixfbecker was referring to is `at-rule-empty-line-before` and not the rule `rule-empty-line-before` that you've fixed in 418af4e. Right now, I still have to add `'at-rule-empty-line-before': null,` to my config to prevent the conflicts reported here. Not a big deal but it will be good to add it to your preset 👍

**Comment by shannonmoeller** [comment]
Thanks, @yhuard! Fixed and published as v3.2.0.
